tables/clb_access_log/clb_access_log.go

```diff
@@ -57,6 +59,16 @@ func (l *ClbAccessLog) InitialiseFromMap(m map[string]string) error {
 					return fmt.Errorf("error parsing client_port: %w", err)
 				}
 			}
+		case "backend":
+			if strings.Contains(value, ":") {
+				parts := strings.Split(value, ":")
+				ip := parts[0]
+				l.BackendIP = &ip
+				l.BackendPort, err = strconv.Atoi(parts[1])
+				if err != nil {
+					return fmt.Errorf("error parsing backend_port: %w", err)
+				}
+			}
```

Comment:
If we know `backend` is always an `ip:port` combination, we could declare that in our parser config as two tokens separated by `:` ?

Reply:
For `backend` field we cannot do that as it can be empty sometimes but for `client` field have split them into `client_ip` and `client_port`